### PR TITLE
implemented isConsonant

### LINKED ISSUE #480: Fix isConsonant function
There is currently a method called `isConsonant` that lives in `/lib/isConsonant.js`.
It is incomplete and needs to be fixed!

This method will return true if the parameter passed is a consonant, if its not it will return false.

### Acceptance Criteria:
1. Running `yarn test isConsonant` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by vincealdrin**
Closes #480 

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: test/pull.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  test("pull [2, 4, 7] from [1, 2, 4, 6, 7] should equal to [1, 6]", () => {
+
+    expect(pull([1, 2, 4, 6, 7], [2, 4, 7])).toEqual([1, 6]);
+  });
+
+  test("pull 23, 7 from [23, 10, 2, 7] should equal to [23, 10]", () => {
+
+    expect(pull([23, 10, 2, 7], 23, 7)).toEqual([23, 10]);
+  });
+
```
Comment (ktilcu) [inline test/pull.test.js]: What should happen if the array has Complex objects like arrays or objects? or if the array is empty or if it isn't an array?

Comment (vincealdrin) [reply test/pull.test.js]: @ktilcu I added more test cases for those

**Comment by ktilcu** [comment]
Thanks for the contribution! It would be really helpful if you could open an issue for the new function you created. Happy Hacktoberfest 🎃 !
